src/Uno.UI.RuntimeTests/Helpers/NumberAssert.cs

```diff
@@ -20,6 +20,17 @@ public static void Greater(double arg1, double arg2)
 				throw new AssertFailedException($"{nameof(arg1)}={arg1} was expected to be greater than {nameof(arg2)}={arg2}");
 			}
 		}
+		/// <summary>
+		/// Asserts that <paramref name="arg1"/> is greater than or equal to <paramref name="arg2"/>.
+		/// </summary>
+		public static void GreaterOrEqual(double arg1, double arg2)
+		{
+			var isGreater = arg1 >= arg2;
+			if (!isGreater)
+			{
+				throw new AssertFailedException($"{nameof(arg1)}={arg1} was expected to be greater than or equal to {nameof(arg2)}={arg2}");
+			}
+		}
```

Comment:
You should use [FluentAssertions](https://fluentassertions.com/numerictypes/) for that kind of comparisons.
``` csharp
measuredValue.Should().BeGreaterOrEqualTo(expectedValue);
```

Reply:
I'll keep that in mind for the future. :)